src/java/htsjdk/samtools/util/Locatable.java

```diff
@@ -22,4 +22,63 @@
      * @return 1-based closed-ended position, undefined if getContig() == null
      */
     int getEnd();
+
+
+    /**
+    * @return the 0-based start position (from the GA4GH spec).
+    */
+    default long getGA4GHStart() {return this.getStart() - 1; }
+
+    /**
+    * @return the typical end spans are [zero-start,end) (from the GA4GH spec).
+    */
+    default long getGA4GHEnd() { return this.getEnd(); }
+
+    /**
+     * @return number of bases covered by this interval (will always be > 0)
+     */
+    default int size() {
```

Comment:
Rename to `getLengthOnReference()` to avoid confusion in implementations of `Locatable` (eg., to avoid the nasty situation of `SAMRecord` having both `length()` and `size()` that mean completely different things.